rsciio/digitalsurf/_api.py

```diff
@@ -1246,14 +1412,17 @@ def _pack_data(self, file, val, encoding="latin-1"):
 
 def file_reader(filename, **kwds):
     """Read a mountainsmap .sur file and return a dictionnary containing the
-    information necessary for creating the data object.
+    information necessary for creating the data object
 
     Parameters
     ----------
-    %s
-    %s
+    filename: name of the .sur file to be read
```

Comment:
Please use the standard strings that we have implemented across RosettaSciIO for easier maintenance (also a reason the tests are currently failing).

Reply:
Ok will look into it

Reply:
This should be taken care of now. Indeed the docstring defaults got lost (I shamelessly admit having performed code pasta from the hyperspy PR)